packages/handlersjs-http/lib/handlers/http-cors-request.handler.spec.ts

```diff
@@ -93,9 +82,11 @@ describe('HttpCorsRequestHandler', () => {
       // Check 'noCorsRequestContext' and 'cleanHeaders'
       it('should not pass CORS headers to the child handler.handle and all headers must be lower case', async() => {
 
+        delete context.request.headers.origin;
```

Comment:
You know I don't like delete ... especially here: you task the reader of the code to know that those headers are set in the beforeEach block, and the delete here thus has no effect on the next test. If this would have been some kind of destructuring, that would have been clear from the start.